src/js/components/DateInput/utils.js

```diff
@@ -125,6 +125,30 @@ const pullDigits = (text, index) => {
   return text.slice(index, end);
 };
 
+export const validateDateBounds = ({ dateBounds, selectedDate }) => {
```

Comment:
```suggestion
export const validateBounds = ({ bounds, selectedDate }) => {
```
This is just a naming recommendation to keep the function naming more concise. Additionally, we try to align with prop names when possible (so simplify to "bounds").

Reply:
sure, fixed 👍🏼